CHANGELOG.md

```diff
@@ -40,6 +40,8 @@ Note that this project **does not** adhere to [Semantic Versioning](http://semve
 - Filenames of external files can no longer contain curly braces. [#5926](https://github.com/JabRef/jabref/pull/5926)
 - We made the filters more easily accessible in the integrity check dialog. [#5955](https://github.com/JabRef/jabref/pull/5955)
 - We reimplemented and improved the dialog "Customize entry types". [#4719](https://github.com/JabRef/jabref/issues/4719)
+- We reimplemented and improved the dialog "Customize entry types" [#4719](https://github.com/JabRef/jabref/issues/4719)
```

Comment:
This is a duplicate.

Reply:
I fixed it in `master`. https://github.com/JabRef/jabref/commit/64afe4ea631b6fe6b10f1d30ad43106bb1a265dc @Siedlerchr